### PR TITLE
In embed model add margins around the sides

### LINKED ISSUE #10878: [CAL-2370] Public booking page embed needs margins around the sides
The embed modal should have a margin on the left and right that matches the margin at the bottom, it should not hug the edge of the page.

CleanShot 2023-08-22 at 08 45 25@2x

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2370](https://linear.app/calcom/issue/CAL-2370/public-booking-page-embed-needs-margins-around-the-sides)</sub>

### PR DESCRIPTION
**Pull request by MeenuyD**
In the embed modal added a margin on the left and right which matches the margin at the bottom.

Fixes #10878

### CODE REVIEW

**Comment by ciaranha** [comment]
@MeenuyD Can you attach a screenshot at various screen sizes or a video recording please?

File: .env.example
Comment (ujwalkumar1995) [inline .env.example:1]: Hey @MeenuyD 
I don't think we are supposed to delete this file as part of this PR.

File: yarn.lock [hunk 1]
```diff
@@ -5,46 +5,6 @@ __metadata:
   version: 6
```
Comment (ujwalkumar1995) [inline yarn.lock:5]: Hey @MeenuyD 
Should not commit yarn.lock file as you have not made any changes to the packages

Comment (MeenuyD) [reply yarn.lock:5]: ok

Comment (MeenuyD) [reply yarn.lock:5]: Should I create new pr

Comment (ujwalkumar1995) [reply yarn.lock:5]: You can fix the changes as part of the same PR. May be you can revert last commit and put another commit with the changes that are only needed as part of the PR. Also, as suggested by @ciaranha , please submit a screenshot or recording of the changes.

**Pull request closed without merging**